lib/diet.ml

```diff
@@ -561,6 +575,20 @@ module Test = struct
 
   let test_depth () = check_depth 1048576
 
+  let test_find_next () =
+    let open IntDiet in
+    let set = add (9, 9) @@ add (5, 7) empty in
+    assert (find_next_gap 0 set = 0);
```

Comment:
I think these `assert`s should be `OUnit2.assert_bool` (which will survive being compiled with `-noassert`)

Reply:
Good point, but I went with the local style.
Should I switch the whole suite to OUnit2 asserts?

Reply:
Yes please, I think that'll be much better

Reply:
ounit?? nowadays you should use alcotest :p

Reply:
Although that would be a bit strange to use ounit/alcotest inside a library. Is there any way to test some of these invariants using the public API ?

Reply:
Changing the test style doesn't have much to do with this pull request.
Also, the invariants seem to rely on internal metadata.
Could this change be punted on for now?

Reply:
yes, that's fine. thanks for the PR!